### PR TITLE
Added documentation for fields in module and updated description as p…

### PR DESCRIPTION
**Pull request by sharadkesarwani**
Added documentation for fields in module and updated description as per apiDoc

# Jira Ticket
Ref: https://spotinst.atlassian.net/browse/SPOTAUT-15257

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,25 +23,25 @@ terraform {
...
+  node_min_count            = 1
+  node_max_count            = 100
+  spot_percentage           = 50
   fallback_to_ondemand      = true
-  tags                      = { "key1": "value1", "key2": "value2" }
-  labels                    = { "labelkey1": "labelvalue1","labelkey2": "labelvalue2"}
-  taints                    = [{"key":"noa","value":"veryMuch", "effect" : "NoSchedule"}]
+  tags                      = { "tagKey": "env", "tagValue": "staging" }
+  labels                    = { "key": "env","value": "test"}
+  taints                    = [{"key":"taintKey","value":"taintValue", "effect" : "NoSchedule"}]
```
Comment (chandra1-n) [inline README.md:51]: add vmSizes usage

File: variables.tf [hunk 2]
```diff
@@ -4,36 +4,36 @@ variable "ocean_vng_name" {
...
   type        = number
   default     = null
-  description = "Optionally configure the number of CPUs to allocate for each headroom unit. CPUs are denoted in millicores, where 1000 millicores = 1 vCPU."
+  description = "Configure the number of CPUs to allocate the headroom. CPUs are denoted in millicores, where 1000 millicores = 1 vCPU."
 }
 variable "headrooms_gpu_per_unit" {
   type        = number
   default     = null
-  description = "Optionally configure the number of GPUS to allocate for each headroom unit."
+  description = "Amont of GPU to allocate for headroom unit."
```
Comment (chandra1-n) [inline variables.tf]: correct the spelling of Amount

**Review by chandra1-n — CHANGES_REQUESTED** [review]
please fix the comments

File: README.md (hunk 1 above)
Comment (sharadkesarwani) [reply README.md:51]: Added

File: variables.tf (hunk 2 above)
Comment (sharadkesarwani) [reply variables.tf]: corrected typo

File: README.md [hunk 3]
```diff
@@ -78,6 +84,49 @@ We use GitHub issues for tracking bugs and feature requests. Please use these co
...
+
+No modules.
+
+## Inputs
+
+| Name | Description | Type | Default | Required |
+|------|-------------|------|---------|:--------:|
+| <a name="input_ocean_vng_name"></a> [ocean\_vng\_name](#input\_ocean\_vng\_name) | Enter a name for the virtual node group. | `string` | `null` | yes |
+| <a name="input_ocean_id"></a> [ocean\_id](#input\_ocean\_id) | The Ocean cluster identifier. Required for Launch Spec creation. | `string` | `null` | yes |
+| <a name="input_headrooms_num_of_units"></a> [headrooms\_num\_of\_units](#input\_headrooms\_num\_of\_units) | The number of units to retain as headroom, where each unit has the defined headroom CPU, memory. | `number` | `null` | no |
```
Comment (chandra1-n) [inline README.md]: change to autoscale_headrooms_

**Review by chandra1-n — CHANGES_REQUESTED** [review]
fix the comments

**Review by chandra1-n — CHANGES_REQUESTED** [review]
also verify the variable names in main.tf file

File: README.md (hunk 3 above)
Comment (sharadkesarwani) [reply README.md]: appended autoscale_ to update variable names.

**Review by chandra1-n — APPROVED** [review]

**Pull request merged**